### DISCUSSION TITLE
I cannot change httpd-vhosts.conf and httpd.conf

### DISCUSSION THREAD

**Discussion by timechips** [nextcloud/all-in-one#4888] [Questions] [answered]
I wish to deploy a simple webpage next to nextcloud with a different domain. I have come to the conclusion that the best way of doing so was editing the `httpd-vhosts.conf` file so that there would be two virtual hosts, then edit the `httpd.conf`. I don't know if this is the right way since I am new to this.
The problem I am facing is that I cannot edit these two files with `sudo docker exec -it nextcloud-aio-apache` and then `vi ./conf/extra/httpd-vhosts.conf`, but no matter what I try I cannot change them because the "filesystem is read only" (I have a dnf docker installation. I have tried --privileged flag and --user root, I tried to copy the file, edit it and copy back, but no luck in any of them)

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
